lib/capnp_c.h

```diff
@@ -22,6 +28,19 @@
 typedef intmax_t ssize_t;
 #endif
 
+// Macro UNUSED(x) silences compiler warnings about unused parameter or variable `x`.
+#ifndef UNUSED
+#define UNUSED(x) (void)(x)
+#endif
+
+// Cross-platform macro ALIGNED_(x) aligns a struct by `x` bits.
```

Comment:
https://msdn.microsoft.com/en-us/library/83ythb65.aspx says it's bytes on MSVC too.

(btw, the comment in capn-malloc.c is wrong too, it shouldn't say `__declspec(align(64))` but actually `__declspec(align(8))`)

For what it's worth - I'm not sure why the alignment (and the check for it) are there to begin with.  Seems to be pretty arbitrary out of some "make this faster" logic (??) - in that case it might be better to remove it actually...

(especially if you are on a microcontroller with little memory, wasting bytes there could be pretty annoying...)